objects/obj_player.object.gmx

```diff
@@ -228,8 +237,10 @@ else if (running &amp;&amp; move != 0) // Check if running
 else
 {       //stay in runjump sprite until below running speed or gliding
     if((run_jump &amp;&amp; abs(hsp) &lt; basemovespeed*runmult) ||(run_jump &amp;&amp; gliding)){ 
-        sprite_index = spr_player; // Set to normal sprite
+        sprite_index = spr_player; // Set to normal sprite and stop long jump
         run_jump =false;
+    }else if(!run_jump){
+        sprite_index = spr_player;//set sprite to normal if there is no long jump

```

Comment:
You can go ahead and remove this, it'll be handled in my changes, and if left in could cause sprite flickering or early reset to walk sprite while still above run speed.